spec/ruby/floating_points_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'floating_points'
+
+RSpec.describe FloatingPoints do
+  it 'parses test properly' do
+    r = FloatingPoints.from_file('src/floating_points.bin')
+    
+    delta = 1e-6
+
+    expect(r.single_value).to be_within(delta).of 1.2345
+    expect(r.single_value_be).to be_within(delta).of 1.2345
+    expect(r.double_value).to be_within(delta).of 123.456
+    expect(r.double_value_be).to be_within(delta).of 123.456
```

Comment:
Ah, I forgot about the internal representation always being double for languages like JS and Ruby. I didn't run into the problem on the C# tests because I was using `1.2345f` to make sure the representation is the same. It's probably not needed for Java or C++ either, they have native support for single precision floats.

I'll tweak the test later to use values that can be represented exactly in a float/double, then a delta won't be needed for any language.